capa/main.py

```diff
@@ -514,7 +512,7 @@ def get_extractor(
     format_: str,
```

Comment:
this function's argument `path` should be a Path not a str, too, right?

Reply:
ideally add types in the function signature (as well)